src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimantRejectOrgPaymentPlanStaffNotificationServiceTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.cmc.claimstore.MockSpringTest;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.claimstore.documents.output.PDF;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.email.EmailAttachment;
+import uk.gov.hmcts.cmc.email.EmailData;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildClaimantResponseFileBaseName;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildResponseFileBaseName;
+import static uk.gov.hmcts.cmc.claimstore.utils.DocumentNameUtils.buildSealedClaimFileBaseName;
+
+public class ClaimantRejectOrgPaymentPlanStaffNotificationServiceTest extends MockSpringTest {
+
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+
+    @Autowired
+    private ClaimantRejectOrgPaymentPlanStaffNotificationService service;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
+    @Captor
+    private ArgumentCaptor<EmailData> emailDataArgument;
+
+    @Autowired
+    private StaffEmailProperties emailProperties;
+
+    private Claim claim;
+
+    @Before
+    public void setup() {
+        claim = SampleClaim.getWithClaimantResponse();
+        when(pdfServiceClient.generateFromHtml(any(byte[].class), anyMap()))
+            .thenReturn(PDF_CONTENT);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void shouldThrowNullPointerWhenGivenNullClaim() {
+        service.notifyStaffClaimantRejectOrganisationPaymentPlan(null);
+    }
+
+    @Test
+    public void shouldSendEmailToExpectedRecipient() {
+        service.notifyStaffClaimantRejectOrganisationPaymentPlan(claim);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
```

Comment:
String captors normally aren't needed and here you're not inspecting the `emailDataArgument` anyway, this can be simplified to
```
verify(emailService).sendEmail(eq(emailProperties.getSender()), any(EmailData.class));
```
and remove the assertion